cms/keystone.js

```diff
@@ -64,7 +64,7 @@ keystone.set("wysiwyg additional options", {
 	// See https://www.tiny.cloud/docs-4x/plugins/link/#link_title
 	link_title: false,
 	target_list: false,
-	paste_as_text: true,
+	paste_word_valid_elements: "p,b,strong,i,em,br,ul,ol,table,tbody,tr,td",
```

Comment:
Does this need `th` too? (no idea if word tables with a heading row come across with proper th's) Also, do we assume li's are OK if we've specified ul and ol?